src/Airship/Types.hs

```diff
@@ -140,6 +147,7 @@ instance MonadBaseControl b m => MonadBaseControl b (Webmachine m) where
 -- | A convenience synonym that writes the @Monad@ type constraint for you.
 type Handler m a = Monad m => Webmachine m a
```

Comment:
Might be remembering we removed it on our branch.

I don't think it should be there, but we have had that conversation before :-) . I'm curious if you're using it anywhere?